_episodes/04-lists.md

```diff
@@ -0,0 +1,318 @@
+---
+title: "Lists"
+teaching: 10
+exercises: 10
```

Comment:
Are we going to allot time for exercises? It looks like time is going to be tight if we do.

Reply:
nope - that is a mistake. I will fix that in the Lists and Dict/Tuples lessons.